public/pages/Rules/components/RuleEditor/RuleEditorContainer.tsx

```diff
@@ -14,7 +14,7 @@ import { RuleEditorFormModel, ruleEditorStateDefaultValue } from './RuleEditorFo
 import { mapFormToRule, mapRuleToForm } from './mappers';
 import { RuleEditorForm } from './RuleEditorForm';
 import { validateRule } from '../../utils/helpers';
-import { errorNotificationToast } from '../../../../utils/helpers';
+import { DataStore } from '../../../../store/DataStore';
 
 export interface RuleEditorProps {
```

Comment:
Should we remove the ruleService prop now?

Reply:
Similarly check in other places as well.

Reply:
All unnecessary rule service imports are removed.